pkg/postgres/pgutils/scan.go

```diff
@@ -1,17 +1,12 @@
 package pgutils
 
 import (
-	"github.com/gogo/protobuf/proto"
 	"github.com/jackc/pgx/v5"
+	"github.com/stackrox/rox/pkg/protocompat"
 )
 
-type unmarshaler[T any] interface {
```

Comment:
nit: to minimize the diff we can alias `protocompat.Unmarshaler` with `unmarshaler`

Reply:
I think the diff is still small enough here.